apache-httpd/apache-wolfssl-post-quantum.patch

```diff
@@ -0,0 +1,39 @@
+--- httpd-2.4.46.orig/modules/ssl/ssl_engine_init.c	2021-11-18 12:40:42.903822554 -0500
```

Comment:
Maybe consider `/modules/ssl/ssl_engine_init.c` instead. Patch had trouble finding the file with that prefix